src/t_hash.c

```diff
@@ -209,25 +210,222 @@ static void hashDictWithExpireOnRelease(dict *d) {
     ebDestroy(&dictExpireMeta->hfe, &hashFieldExpireBucketsType, NULL);
 }
 
+/*-----------------------------------------------------------------------------
+ * listpackTTL functions
+ *----------------------------------------------------------------------------*/
```

Comment:
I think should follow similar naming convention between `listpack` and `HT`.
Maybe you can use `listpcakEx` instead. WDYT? 
We can leave this decision to the end of this PR.

Reply:
sounds okay to me. We can call it `OBJ_ENCODING_LISTPACK_EX` and `struct listpackEx` etc..